File: Fibonacci.java
public class Fibonacci {

    public static void main(String[] args) {

        int n = 10; // número de termos que queremos

        int a = 0;
        int b = 1;

        System.out.print(a + " " + b);

        for (int i = 2; i < n; i++) {

            int c = a + b;
            System.out.print(" " + c);

            a = b;
            b = c;

        }

    }
}

File: Fibonacci.js
// função que retorna o número fibonacci para determinado valor de n
function fibonacci(n) {
    if (n <= 1) {
    return n;
    } else {
    return fibonacci(n - 1) + fibonacci(n - 2);
    }
    }
    
    // chamada da função para imprimir os 10 primeiros números fibonacci
    for (var i = 0; i < 10; i++) {
    console.log(fibonacci(i));
    }